sds/src/event.rs

```diff
@@ -43,9 +47,90 @@ impl Event for String {
     }
 }
 
+impl Event for serde_json::Value {
```

Comment:
I believe it would make sense to create an event_json.rs instead of pushing everything in the same file no ?

Reply:
[move Event implementation for Hashmap to event_json](https://github.com/DataDog/dd-sensitive-data-scanner/pull/145/commits/335fc5f3496de127c87456a1490fff24274e939b)